docs/how-to/extract-a-backup.md

```diff
@@ -45,6 +45,22 @@ in the right place before running the command—or see below about the
 `--destination` flag.
 
 
+## Extract the configuration file used to create an archive
+
+<span class="minilink minilink-addedin">New in version 1.7.15</span> Borgmatic
+also stores the configuration file used to create an archive, inside the
+archive itself. This is useful in cases where you've lost your configuration
+file, or you want to see what configuration was used to create a particular
+archive.
+
+To extract the configuration file from an archive, use the `config bootstrap` action:
```

Comment:
In this whole section, is it worth pluralizing this and talking about extracting "configuration file**s**" wherever they're mentioned? That seems like an important (and cool!) part of the feature.